### PR TITLE
fix: do not break logo during generation

### PR DESCRIPTION
**Pull request by ctamisier**
Do not break logo during generation (do the same than in Angular)

### CODE REVIEW

File: generators/client/files-react.js [hunk 1]
```diff
@@ -32,11 +32,11 @@ const files = {
                 '.eslintrc.json',
                 'tsconfig.json',
                 'tsconfig.test.json',
-                'webpack/logo-jhipster.png',
                 'webpack/webpack.common.js',
                 'webpack/webpack.dev.js',
                 'webpack/webpack.prod.js',
                 'webpack/utils.js',
+                { file: 'webpack/logo-jhipster.png', method: 'copy' }
```
Comment (qmonmert) [inline generators/client/files-react.js:39]: `npm run lint-fix` to fix error, I think comma is missing at the end of the line

Comment (ctamisier) [reply generators/client/files-react.js]: oh yes, thx !

Comment (qmonmert) [reply generators/client/files-react.js]: 😉

**Review by mshima — APPROVED** [review]
LGTM

**Comment by mshima** [comment]
Thanks @ctamisier

**Pull request merged**